src/main/java/socialite/logic/commands/AddCommand.java

```diff
@@ -27,26 +27,26 @@
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + "[" + PREFIX_TAG + "TAG]... "
+            + "[" + PREFIX_DATE + "NAME:YYYY-MM-DD[:monthly|:yearly]]... "
             + "[" + PREFIX_FACEBOOK + "FACEBOOK] "
             + "[" + PREFIX_INSTAGRAM + "INSTAGRAM] "
             + "[" + PREFIX_TELEGRAM + "TELEGRAM] "
             + "[" + PREFIX_TIKTOK + "TIKTOK] "
-            + "[" + PREFIX_TWITTER + "TWITTER] "
-            + "[" + PREFIX_DATE + "NAME:YYYY-MM-DD]...\n"
+            + "[" + PREFIX_TWITTER + "TWITTER]\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_TAG + "friends "
             + PREFIX_TAG + "owesMoney "
+            + PREFIX_DATE + "birthday:2000-01-01 "
```

Comment:
Maybe we can add `:yearly` for the birthday here?

Reply:
Alright! Will add it now heh